src/elife_profile/elife_profile.make.yml

```diff
@@ -201,6 +201,8 @@ projects:
     version: '1.0-rc2'
   services:
     version: '3.12'
+    patch:
+        - 'https://gist.githubusercontent.com/giorgiosironi/d5f3817161f7a38d457b4fd1eb5b2c75/raw/1a5bade0e8307a77066b01323f479c54889d8478/drupal_rest_server_exception_logging.patch'
```

Comment:
The problem is general: exceptions are transformed in 500 errors but do not get logged. The solution however is blunt, just `error_log()` everything.